$OpenBSD: patch-src_libpiano_piano_h,v 1.4 2012/06/08 11:01:20 dcoppa Exp $

Fix ambiguous error message "Invalid Partner Login" on wrong
email address/password failure
(upstream git commit 81bf363ac69e0036562434ca0e7153c3e624c8c3)

More error messages
(upstream git commit b32348d88f700f4a3b5fb3a95f7ee40336952230)

--- src/libpiano/piano.h.orig	Sun May  6 16:33:54 2012
+++ src/libpiano/piano.h	Fri Jun  8 12:38:08 2012
@@ -244,7 +244,9 @@ typedef enum {
 	PIANO_RET_INVALID_RESPONSE = 2,
 	PIANO_RET_CONTINUE_REQUEST = 3,
 	PIANO_RET_OUT_OF_MEMORY = 4,
+	PIANO_RET_INVALID_LOGIN = 5,
 
+	/* pandora error codes */
 	PIANO_RET_P_INTERNAL = PIANO_RET_OFFSET+0,
 	PIANO_RET_P_API_VERSION_NOT_SUPPORTED = PIANO_RET_OFFSET+11,
 	PIANO_RET_P_BIRTH_YEAR_INVALID = PIANO_RET_OFFSET+1025,
@@ -284,7 +286,7 @@ typedef enum {
 	PIANO_RET_P_URL_PARAM_MISSING_USER_ID = PIANO_RET_OFFSET+5,
 	PIANO_RET_P_USERNAME_ALREADY_EXISTS = PIANO_RET_OFFSET+1013,
 	PIANO_RET_P_USER_ALREADY_USED_TRIAL = PIANO_RET_OFFSET+1037,
-	PIANO_RET_P_USER_NOT_ACTIVE = PIANO_RET_OFFSET+1003,
+	PIANO_RET_P_LISTENER_NOT_AUTHORIZED = PIANO_RET_OFFSET+1003,
 	PIANO_RET_P_USER_NOT_AUTHORIZED = PIANO_RET_OFFSET+1004,
 	PIANO_RET_P_ZIP_CODE_INVALID = PIANO_RET_OFFSET+1024,
 
